cmd/git-init/main.go

```diff
@@ -25,19 +25,27 @@ import (
 )
 
 var (
-	url                    = flag.String("url", "", "The url of the Git repository to initialize.")
-	revision               = flag.String("revision", "", "The Git revision to make the repository HEAD")
-	path                   = flag.String("path", "", "Path of directory under which git repository will be copied")
-	terminationMessagePath = flag.String("terminationMessagePath", "/dev/termination-log", "Location of file containing termination message")
-	submodules             = flag.Bool("submodules", true, "Initialize and fetch git submodules")
+	fetchSpec              git.FetchSpec
+	submodules             bool
+	terminationMessagePath string
 )
 
+func init() {
```

Comment:
nit: any reason for this to be in `init()` and not on top of `main()` ; the end result should be the same :)

Reply:
I find this more intuitive than sticking it in `main`. Most uses of the `flag` package I’ve seen don’t use the `…Var` forms and just live inside a top-level `var` block. Those calls run before `main` and stand out visually from the code in `main` – to me, this felt like a more consistent and minimal refactor than adding them to `main`.